curriculum/js0/4.js

```diff
@@ -5,7 +5,10 @@
  */
 
 const solution = (a, b) => {
-  return 0;
+  if (a > b) {
+    return a;
+  }
+  return b;
```

Comment:
This works. In case you're interested, here are a few alternative ways to write it:

* the brackets can be dropped if the statement is not more than one line:
```javascript
if (a > b) return a;
return b;
```
* we can also use [the ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator):
```javascript
return a > b ? a : b;
```